src/lv_objx/lv_chart.c

```diff
@@ -401,15 +401,14 @@ void lv_chart_set_next(lv_obj_t * chart, lv_chart_series_t * ser, lv_coord_t y)
 }
 
 /**
- * Set the x-axis label of a chart
- * @param chart pointer to a chart object
- * @param text  pointer to the label text (not copied)
+ * Set the margin around the chart, used for axes value and labels
+ * @param margin	value of the margin
  */
-void lv_chart_set_label(lv_obj_t* chart, const char* text)
+void lv_chart_set_margin(lv_obj_t* chart, uint16_t margin)
```

Comment:
For consistency there should be a `lv_chart_get_margin` function too.

Reply:
Fixed.

Reply:
I don't like being picky like this, but the documentation for the `get` functions still has `Set` in it.

Reply:
My mistake.